contracts/MarketOracle.sol

```diff
@@ -48,6 +50,13 @@ contract MedianOracle is Ownable, IOracle {
         reportExpirationTimeSec = reportExpirationTimeSec_;
     }
 
+    function setReportSecurityDelaySec(uint256 reportSecurityDelaySec_)
+    external
+    onlyOwner
+    {
+        reportSecurityDelaySec = reportSecurityDelaySec_;
```

Comment:
Do we need to check that security delay is < expiration time?

Logically it doesn't make much sense... not sure if our logic would break if it's broken.

Reply:
if reportSecurityDelaySec > reportExpirationTimeSec all reports timestamp will be considered out of range, but nothing will break. I rather not check it explicitly. TO not care about order of operation if changing both of them in one go.